samples/echobot/main.go

```diff
@@ -13,6 +13,12 @@ import (
 
 var Adapter *adapter.BotFrameworkAdapter
 
+var customHandler = adapter.ActivityHandlerFuncs{
+	MessageFuntion: func(turn *adapter.TurnContext) interface{} {
```

Comment:
Can we use format: `On<activityType>Func`, like `OnMessageFunc` or `OnTypingFunc`?

Reply:
Changed the name.